src/GitVersionExe/ArgumentParser.cs

```diff
@@ -52,7 +54,7 @@ public static Arguments ParseArguments(List<string> commandLineArguments)
 
             List<string> namedArguments;
             var arguments = new Arguments();
-            if (firstArgument.StartsWith("-") || firstArgument.StartsWith("/"))
+            if (firstArgument.StartsWith("-") || (firstArgument.StartsWith("/") && Path.DirectorySeparatorChar != '/'))
```

Comment:
FYI we are probably going to redo the command line args in 4.0.0, which should help cross plat out as we will probably drop `/` arguments